index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   agent:
+  - apiVersion: v2
+    appVersion: 3.2.0
+    created: "2023-12-19T19:15:40.686472+06:00"
+    description: Helm chart for the Optimizely Agent https://docs.developers.optimizely.com/full-stack-experimentation/docs/optimizely-agent
+    digest: 964d48b55bf52809b78bce5a81cee28fd57b432f39880c3ef31cb1f67aa1c847
+    home: https://docs.developers.optimizely.com/full-stack-experimentation/docs/optimizely-agent
+    icon: https://raw.githubusercontent.com/optimizely/optimizely-agent-helm/main/Optimizely-icon.png
+    keywords:
+    - Optimizely
+    - Optimizely Agent
+    - Full Stack
+    - Rollouts
+    - Feature Flags
+    maintainers:
+    - email: devx@optimizely.com
+      name: Optimizely Dev X Team
+    name: agent
+    sources:
+    - https://github.com/optimizely/agent
+    type: application
+    urls:
+    - https://optimizely.github.io/optimizely-agent-helm/packages/agent-1.3.0.tgz
+    version: 1.3.0
```

Comment:
Is this the Chart version?

Reply:
For consideration: Perhaps it's worth creating a [Release](https://github.com/optimizely/optimizely-agent-helm/releases) to show the current version 🤷 just 🤔

Reply:
I think it is good to have a matching chart version for the app version upgrade. Otherwise, users have to manually update their deployment/image version as the older chart is pointing to the older version of Agent.